src/KOKKOS/collide_vss_kokkos.cpp

```diff
@@ -626,7 +626,7 @@ void CollideVSSKokkos::operator()(TagCollideCollisionsOne< NEARCP, ATOMIC_REDUCT
   // attempt = exact collision attempt count for a pair of groups
   // nattempt = rounded attempt with RN
 
-  const double attempt = attempt_collision_kokkos(icell,np,volume,rand_gen);
+  const double attempt = attempt_collision_kokkos(icell,np,volume,dt,rand_gen);
```

Comment:
Maybe I am misunderstanding, but it looks like these changes to add `dt` into the functions is not needed?